build-tools/packages/build-cli/docs/publish.md

```diff
@@ -0,0 +1,43 @@
+`flub publish`
+==============
+
+Publish commands are used to publish packages to an npm registry.
+
+* [`flub publish tarballs`](#flub-publish-tarballs)
+
+## `flub publish tarballs`
+
+Publishes tarballs to the package registry unless the version is already published.
+
+```
+USAGE
+  $ flub publish tarballs [-v | --quiet] [--dir <value>] (--tarball --orderFile <value>) [--retry <value>] [--dryRun]
```

Comment:
I don't understand the use of `( ... )` here. It looks like you must use --tarball to use --orderFile (noted as required). What is it supposed to mean?

Reply:
It's because `--tarball` depends on `--orderFile`. Even though orderFile is required right now, I envision relaxing that at some point and accepting a list from stdin or some other source. I can remove the constraint since orderFile is required for now.